### DISCUSSION TITLE
Submit your story maps here

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21W-DH140#46] [Final presentations]
Submit your story map link, along with your group name, members of your group, and links to your Jupyter Notebooks in your JupyterHub, and links to your jupyter notebooks in your GitHub repo.

Example:
**Group Name**: Spatial Guru

**Group Members**: Yoh and his cat

**Story map link**: [https://arcg.is/1n0vGi](https://arcg.is/1n0vGi)

**Jupyter Notebook Link**: [https://jupyter.idre.ucla.edu/user/yohman@gmail.com/notebooks/instructor/21W-DH140/Weeks/Week10/W10-01-Publishing%20interactive%20visualizations.ipynb](https://jupyter.idre.ucla.edu/user/yohman@gmail.com/notebooks/instructor/21W-DH140/Weeks/Week10/W10-01-Publishing%20interactive%20visualizations.ipynb)

**GitHub Link**: [https://github.com/yohman/21W-DH140/blob/master/Weeks/Week10/W10-01-Publishing%20interactive%20visualizations.ipynb](https://github.com/yohman/21W-DH140/blob/master/Weeks/Week10/W10-01-Publishing%20interactive%20visualizations.ipynb)

**Comment by emdipressi** [comment]
**Group Name:** Education and Crime 

**Group Members:** Emily DiPressi and Tiffany Feng 

**Story map link:** https://storymaps.arcgis.com/stories/4bf94875de01452c9018c017de69ee3f 

**Jupyter Notebook Links:** Spending on Education Visualizations:
https://jupyter.idre.ucla.edu/user/tiffanyfeng456@gmail.com/notebooks/21W-DH140/Tiffany%20Feng%20-%20Midterm.ipynb

Tweets - North Dakota:
https://jupyter.idre.ucla.edu/user/tiffanyfeng456@gmail.com/notebooks/21W-DH140/Week%207%20%20Assignment%20-%20Tiffany%20Feng.ipynb

Moran’s Plot & Spatial Autocorrelation:
https://jupyter.idre.ucla.edu/user/tiffanyfeng456@gmail.com/notebooks/21W-DH140/Group%20Assignment%20%234%20Moran's%20Plot%20and%20Spatial%20Autocorrelation.ipynb 

Hate Crime Viz:
https://jupyter.idre.ucla.edu/user/edipressi@ucla.edu/notebooks/21W-DH140/Individual%20Assignments/Final--Emily%20DiPressi.ipynb 

Tweets - District of Columbia:  
https://jupyter.idre.ucla.edu/user/edipressi@ucla.edu/notebooks/21W-DH140/Individual%20Assignments/Week%207%20Assignment%20--%20Emily%20DiPressi%20.ipynb 

**GitHub Group Repo Links:** https://github.com/emdipressi/DH-140-Project/tree/main/Final%20Jupyter%20Notebooks 
Spending on Education Visualizations: https://github.com/emdipressi/DH-140-Project/blob/main/Final%20Jupyter%20Notebooks/Spending%20on%20Education%20Visualizations.ipynb 

Tweets - North Dakota: https://github.com/emdipressi/DH-140-Project/blob/main/Final%20Jupyter%20Notebooks/Tweets%20North%20Dakota.ipynb

Moran’s Plot & Spatial Autocorrelation: https://github.com/emdipressi/DH-140-Project/blob/main/Final%20Jupyter%20Notebooks/Morans%20Plot%20and%20spatial%20autocorr.ipynb

Hate Crime Visualizations: https://github.com/emdipressi/DH-140-Project/blob/main/Final%20Jupyter%20Notebooks/Final--Emily%20DiPressi.ipynb 

Tweets - District of Columbia: https://github.com/emdipressi/DH-140-Project/blob/main/Final%20Jupyter%20Notebooks/Week%207%20Assignment%20--%20Emily%20DiPressi%20%20(1).ipynb

**Comment by derekwang99** [comment]
**Group Name: Social and Economic Determinants of Community Health

**Group Members:** Derek Wang and Henry Khong

**Story map link:** https://arcg.is/fHmDC

**Jupyter Notebook Link:**
City Comparisons: https://jupyter.idre.ucla.edu/user/henrykhong@g.ucla.edu/notebooks/21W-DH140/Personal%20Space%20💆🏻/HPI_City_Comparison.ipynb#

Essential Businesses & HPI Choropleth Map: https://jupyter.idre.ucla.edu/user/derekwang250622@gmail.com/notebooks/21W-DH140/Essential%20Businesses%20and%20HPI%20Choropleth.ipynb

Spatial Autocorrelation Analysis: https://jupyter.idre.ucla.edu/user/derekwang250622@gmail.com/notebooks/21W-DH140/Spatial%20Autocorrelation%20Notebook.ipynb

**GitHub Link:** https://github.com/derekwang99/DH140_Group_Project/tree/main/Final

HPI City Comparisons: https://github.com/derekwang99/DH140_Group_Project/blob/main/Final/HPI_City_Comparison.ipynb

Essential Businesses & HPI Choropleth Map: https://github.com/derekwang99/DH140_Group_Project/blob/main/Final/Essential%20Businesses%20and%20HPI%20Choropleth.ipynb

Spatial Autocorrelation Analysis: https://github.com/derekwang99/DH140_Group_Project/blob/main/Final/Spatial%20Autocorrelation%20Notebook.ipynb

**Comment by claireg22** [comment]
**Group Name:** Global Music Trends

**Group Members:** Claire Guo and Pawan(Sine) Polcharoen 

**Story map link:** https://storymaps.arcgis.com/stories/ebc927c6fb984b5bb8e61bca236fa62c

**Jupyter Notebook Link:** 
Quartile Analysis:
https://jupyter.idre.ucla.edu/user/claireguo123@gmail.com/notebooks/Quartile%20Analysis%20-%20Final.ipynb#

Spatial Autocorrelation:
https://jupyter.idre.ucla.edu/user/spolcharoen@gmail.com/notebooks/DH140/DH140%20Final%20Spatial%20Autocorrelation.ipynb#

**GitHub Link:** https://github.com/spolcharoen/DH140_Project
Quartile Analysis:
https://github.com/spolcharoen/DH140_Project/blob/main/Quartile%20Analysis%20-%20Final.ipynb

Spatial Autocorrelation:
https://github.com/spolcharoen/DH140_Project/blob/main/DH140%20Final%20Spatial%20Autocorrelation.ipynb

**Comment by paigedefiori** [comment]
Group Name: COVID-19 Vaccine

Group Members: Paige Defiori and Alessandro Hammond

Story map link: https://arcg.is/1Cejqj0

Jupyter Notebook Link: 
https://jupyter.idre.ucla.edu/user/paigedefiori@g.ucla.edu/notebooks/21W-DH140/Midterm%20and%20Finals/Final%20Notebook.ipynb

https://jupyter.idre.ucla.edu/user/paigedefiori@g.ucla.edu/notebooks/21W-DH140/Weeks/Week09/Twitter%20Analysis.ipynb


GitHub Link: 
https://github.com/paigedefiori/Group-Project/blob/main/Group%20Assignments/Final%20Notebook.ipynb 

https://github.com/paigedefiori/Group-Project/blob/main/Group%20Assignments/Twitter%20Analysis.ipynb

**Comment by arielhanasab** [comment]
**Group Name:** S&P 500

**Group Members:** Ariel Hanasab

**Story map link:** https://storymaps.arcgis.com/stories/3c6a7d51ce1c40348228598019c6a384

**Jupyter Notebook Link:** https://storymaps.arcgis.com/stories/3c6a7d51ce1c40348228598019c6a384

**GitHub Link:** https://storymaps.arcgis.com/stories/3c6a7d51ce1c40348228598019c6a384

**S&P 500 companies by market capitalization:** https://arielhanasab.github.io/Project/Group%20Assignments/S&P_Companies_by_Market_Cap.html

**Huge S&P 500 companies by market cap:** https://arielhanasab.github.io/Project/Group%20Assignments/Huge_Companies_by_Market_Cap.html

**Small S&P 500 companies by market cap:** https://arielhanasab.github.io/Project/Group%20Assignments/Small_Companies_by_Market_Cap.html

**S&P 500 companies on US map:** https://arielhanasab.github.io/Project/Group%20Assignments/Companies_by_Market_Cap_geo.html

**Comment by frida-barragan** [comment]
**Group Name:** Healthcare Availability Exploration in LA County

**Group Members:** Keven Michel and Frida Barragan

**Story map link:** https://storymaps.arcgis.com/stories/9f5648663c374ba5a62c0f470fc87bd7

**Jupyter Notebook Link:**
 
_Frida Barragan Notebook:_ https://jupyter.idre.ucla.edu/user/nicole.barragan@icloud.com/notebooks/21W-DH140/Final_Project_Frida_Barragan.ipynb

_Keven Michel Notebook:_ https://jupyter.idre.ucla.edu/user/michelkeven15@gmail.com/notebooks/21W-DH140/Keven/DH140_Final_Keven.ipynb#

**GitHub Links:**

 _Frida Barragan Link:_ https://github.com/frida-barragan/Group-Assignments/blob/main/Final_Project_Frida_Barragan.ipynb

_Keven Michel Link:_ https://github.com/frida-barragan/Group-Assignments/blob/2b702d43edeb3200dbf7c1252664382661b4cf81/DH140_Final_Keven.ipynb

**Comment by yjwsb233** [comment]
Project Name: A Dive into Coral Bleaching
Group member: Haiqi Zhou
Storymap link: https://arcg.is/1rze1b
Jupyter Notebook link: https://jupyter.idre.ucla.edu/user/haiqi@ucla.edu/notebooks/21W-DH140/Weeks/Week04/Final%20project.ipynb
Github link: https://github.com/yjwsb233/dh140-haiqi/blob/main/Final%20project.ipynb

**Comment by hkarchmer** [comment]
**Group Name:** Affordable Housing Projects and Environmental Risk in Los Angeles
**Group Members:** Haley Karchmer and Richard Garcia
**Story Map link:** https://sites.google.com/view/dh140final-garciakarchmer/home     
**Jupyter Notebook Links** 
Exploratory Data Analysis (CES Score/Disadvantaged Communities bar chart, Affordable housing point map using OpenStreetMap): https://jupyter.idre.ucla.edu/user/richardrbg@gmail.com/notebooks/21W-DH140/Midterm%20DH140.ipynb 

Twitter Analysis - #affordablehousing: https://jupyter.idre.ucla.edu/user/hkarchmer@g.ucla.edu/notebooks/twitter/Group%20Assignment%20%233%20Twitter%20Data.ipynb 

Moran's Plot, Spatial Autocorrelation Analysis and Chloropleth Map: https://jupyter.idre.ucla.edu/user/richardrbg@gmail.com/notebooks/21W-DH140/Final%20spatial%20AC.ipynb#  

**GitHub Group Repo Links** 
Exploratory Data Analysis (CES Score/Disadvantaged Communities bar chart, Affordable housing point map using OpenStreetMap): https://github.com/richardrbg/DH-140-Group-Project/blob/main/Midterm%20DH140.ipynb 

Moran's Plot, Spatial Autocorrelation Analysis and Choropleth Map:  https://github.com/richardrbg/DH-140-Group-Project/blob/main/Final%20spatial%20AC.ipynb  

Twitter Analysis – #affordablehousing:
https://github.com/richardrbg/DH-140-Group-Project/blob/main/Group%20Assignment%20%233%20Twitter%20Data%20(1).ipynb

**Comment by jujubee413** [comment]
Group Name: Olympics Committee
Group Members: Adam Kaba and Juliette Oliver
Story Map Link: https://storymaps.arcgis.com/stories/39ef5f5fb73d4d8987aa2af3a70347ef
Final Compiled Jupyter Notebook with all visualizations: https://jupyter.idre.ucla.edu/user/jujubee413@yahoo.com/notebooks/Final.ipynb#

Final Compiled Notebook with all visualizations on Github:
https://github.com/jujubee413/DH140_group_project/blob/main/Final.ipynb

Github Link: https://github.com/jujubee413/DH140_group_project
Individual Github Links:
Juliette- https://github.com/jujubee413/DH140_group_project
Adam- https://github.com/arkaba/dh140

Individual Notebooks:
Group Assignment 2: Census Data Analysis-  Choropleth Maps and Economic Cost Line graph
https://jupyter.idre.ucla.edu/user/jujubee413@yahoo.com/notebooks/DH_wk4_Adam_return.ipynb

Group Assignment 3: Twitter Analysis https://jupyter.idre.ucla.edu/user/jujubee413@yahoo.com/notebooks/WK_7.ipynb

Group Assignment 4: Spatial Analysis- Arc Kepler Graph of Female Representation in the Olympics
https://jupyter.idre.ucla.edu/user/jujubee413@yahoo.com/notebooks/Group%20Project%204.ipynb

**Comment by stacy-carlos** [comment]
**Group Name:** Gun Violence Data
**Group Members:** Stacy Carlos & Ainsleigh Douglas
**Story map link:**  https://storymaps.arcgis.com/stories/65c952077ac640a9a4baaaccf9f6bb2c

**Jupyter Notebook Link:**
Twitter sentiment Analysis:
https://jupyter.idre.ucla.edu/user/ainsleigh.douglas@gmail.com/notebooks/Twitter%20Sentiment%20Analysis%20-%20Illinois.ipynb
https://jupyter.idre.ucla.edu/user/scv143@ucla.edu/notebooks/21W-DH140/Texas-Twitter%20Sentiment%20Analysis.ipynb

Income & Education:
https://jupyter.idre.ucla.edu/user/scv143@ucla.edu/notebooks/21W-DH140/Income%20x%20Education-%20Texas%20%26%20Illinois.ipynb#

Gun violence clusters:
https://jupyter.idre.ucla.edu/user/ainsleigh.douglas@gmail.com/notebooks/Illinois%20-%20Gun%20Violence%20Clusters.ipynb
https://jupyter.idre.ucla.edu/user/ainsleigh.douglas@gmail.com/notebooks/Texas%20-%20Gun%20Violence%20Clusters.ipynb
Spatial Autocorrelation analysis:
https://jupyter.idre.ucla.edu/user/ainsleigh.douglas@gmail.com/notebooks/Texas%20-%20Overlays%20%26%20Spatial%20Correlation%20Analysis-Copy1.ipynb#

https://jupyter.idre.ucla.edu/user/ainsleigh.douglas@gmail.com/notebooks/Illinois%20-%20Overlays%20%26%20Spatial%20Correlation%20Analysis-Copy1.ipynb#


GitHub Link: https://github.com/ainsleighdouglas/Gun-Violence-DH140/tree/main/Final
Twitter sentiment Analysis:
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Twitter%20Sentiment%20Analysis%20-%20Illinois.ipynb

https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Texas-Twitter%20Sentiment%20Analysis.ipynb
Income & Education:
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Income%20x%20Education-%20Texas%20%26%20Illinois.ipynb
Gun violence clusters:
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Illinois%20-%20Gun%20Violence%20Clusters.ipynb
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Texas%20-%20Gun%20Violence%20Clusters.ipynb
Spatial Autocorrelation analysis:
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Illinois%20-%20Spatial%20AutoCorrelation%20Analysis.ipynb
https://github.com/ainsleighdouglas/Gun-Violence-DH140/blob/main/Final/Texas%20-%20Spatial%20AutoCorrelation%20Analysis.ipynb

**Comment by wdnmd97** [comment]
Group Name: Vaccine Ninja

Group Members: Kai Watanabe & Yueshuwei Wu (Jack)

Story map link: https://storymaps.arcgis.com/stories/2f4a17c6ac2049b0b7f23bad6532971f

Jupyter Notebook Link: https://github.com/wdnmd97/DH140_Group_Project/blob/main/Group%20Assignments/Final%20Project.ipynb

GitHub Link: https://github.com/wdnmd97/DH140_Group_Project

**Comment by matthewlach** [comment]
Group Name: EnviroJustice
Group Members: Matthew Lach

Story Map Link: https://sites.google.com/g.ucla.edu/dh140-envirojustice/home#h.a7pqjutrpi14

Jupyter Notebook Link:
https://jupyter.idre.ucla.edu/user/matthewdlach@outlook.com/notebooks/Final%20Notebook%201.ipynb

https://jupyter.idre.ucla.edu/user/matthewdlach@outlook.com/notebooks/Final%20Notebook%202.ipynb

https://jupyter.idre.ucla.edu/user/matthewdlach@outlook.com/notebooks/Income%20Choropleth%20Bubble%20Map.ipynb

GitHub Link:
Final Notebook 1 (Income Analysis)
https://github.com/matthewlach/dh140project/blob/main/Final%20Notebook%201%20(1).ipynb
Final Notebook 2 (Race Analysis)
https://github.com/matthewlach/dh140project/blob/main/Final%20Notebook%202.ipynb
Income Choropleth Bubble Map
https://github.com/matthewlach/dh140project/blob/main/Income%20Choropleth%20Bubble%20Map.ipynb

**Comment by stephchang** [comment]
**Group Name:** Food Insecurity

**Group Members:** Ethan Vuong & Stephanie Chang

**Story map link:** https://storymaps.arcgis.com/stories/a1fec95fe2104c879653e3cf0c3dfcc3?fbclid=IwAR3-ns9gtctKLjj_R-ii1Y7CEK-nFcnztC4mXKmbxhDN2GXYuCtnQXmkZyo

**Jupyter Notebook Links:**

SNAP Across the United States: https://jupyter.idre.ucla.edu/user/stephcha7@g.ucla.edu/notebooks/21W-DH140/Weeks/Week04/1%20SNAP%20Usage%20Per%20State.ipynb

Poverty Rates: https://jupyter.idre.ucla.edu/user/stephcha7@g.ucla.edu/notebooks/21W-DH140/Weeks/Week04/2%20Poverty%20Rates%20Per%20State.ipynb#

Political Affiliation for Oregon: https://jupyter.idre.ucla.edu/user/stephcha7@g.ucla.edu/notebooks/21W-DH140/Final/Political%20Affiliation%20vs%20Snap%20Usage%20-%20Oregon.ipynb

Political Affiliation for Wyoming: https://jupyter.idre.ucla.edu/user/stephcha7@g.ucla.edu/notebooks/21W-DH140/Final/Political%20Affiliation%20vs%20Snap%20Usage%20-%20Wyoming.ipynb

Twitter Analysis: https://l.facebook.com/l.php?u=https%3A%2F%2Fjupyter.idre.ucla.edu%2Fuser%2Fethaanvuong24%40g.ucla.edu%2Fnotebooks%2F21W-DH140%2FWeeks%2FWeek07%2FNewest%2520Twitter%2520Data.ipynb%3Ffbclid%3DIwAR2i-bF3oxF-2gEH03o2SGB0oNyHaH56HFLxFaVEWFqSskLK_0jhQCxyfSU&h=AT12sO_1YhwKqzcR-ojX1c8FUGS7dq_516aoxpJmvEXRow4OlMto2NdTGsqyBCgBsTQcS2Khv1RrOqDdJR5sQ-8WoQAgoEtxC8rmQ-Rv8CBaN-OrDYPM32Q-UxGJKdcq-l1kN9MFwaMuLSF42tX-wodOJJg

SNAP locations: https://jupyter.idre.ucla.edu/user/ethaanvuong24@g.ucla.edu/notebooks/21W-DH140/SNAP%20Store%20Locations.ipynb#
https://jupyter.idre.ucla.edu/user/ethaanvuong24@g.ucla.edu/notebooks/21W-DH140/SNAP%20Store%20Locations.ipynb#

Demographic: https://jupyter.idre.ucla.edu/user/ethaanvuong24@g.ucla.edu/notebooks/21W-DH140/Black%20and%20White%20communities%20below%20poverty%20line.ipynb

**Github links: (all in the folder)**

https://github.com/ethanvuong/Food-insecurity/tree/main/Group%20Assignments/final

**Comment by hoz-map** [comment]
**Group Name**: Disaster

**Group Members**: Hozumi Oshida

**Story map link**: https://arcg.is/19XH1W0

**Jupyter Notebook Links**:
Disaster Trends, Death Toll, Population Increase:
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/Disaster%20Trends%2C%20Death%20Toll%2C%20Population%20Increase.ipynb

Twitter Analysis:
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/Twitter%20Analysis.ipynb

Disaster Point Analysis and Flood Rankings:
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/Disaster%20Point%20Analysis%20and%20Flood%20Rankings.ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Jakarta 2019):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Jakarta%202019).ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Jakarta 2020):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Jakarta%202020).ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Bandung 2019):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Bandung%202020)-Copy1.ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Bandung 2020):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Bandung%202020).ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Minahasa Selatan 2019):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Minahasa%20Selatan%202019)-Copy1.ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Minahasa Selatan 2020):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Minahasa%20Selatan%202020).ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Sumba Barat Daya 2019):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Sumba%20Barat%20Daya%202019).ipynb

NDWI of 2020 Indonesia Floods using Sentinel-2 (Sumba Barat Daya 2020):
https://jupyter.idre.ucla.edu/user/noir0315@g.ucla.edu/notebooks/21W-DH140/NDWI%20of%202020%20Indonesia%20Floods%20using%20Sentinel-2%20(Sumba%20Barat%20Daya%202020).ipynb

**Github links**: 
https://github.com/hoz-map/DH140-project
